### PR TITLE
Removed some unused fields

### PR DESCRIPTION
**Pull request by IvanNardi**

### CODE REVIEW

File: python/ndpi_typestruct.py [hunk 1]
```diff
@@ -229,7 +229,6 @@ class CustomCategories(Structure):
     ("detection_bitmask", NDPIProtocolBitMask),
     ("generic_http_packet_bitmask", NDPIProtocolBitMask),
```
Comment (utoni) [inline python/ndpi_typestruct.py]:
```suggestion
```

**Review by utoni — CHANGES_REQUESTED** [review]

File: python/ndpi_typestruct.py (hunk 1 above)
Comment (IvanNardi) [reply python/ndpi_typestruct.py]: Thanks for spotting that! Python bindings are a completely mess...
Fixed

Comment (utoni) [reply python/ndpi_typestruct.py]: True. We might need some sort of automation there.

**Review by utoni — APPROVED** [review]

**Comment by lucaderi** [comment]
@aouinizied I am going to merge this code but your feedback on Python bindings is welcome

**Pull request merged**